src/state_machine/coordinator/mod.rs

```diff
@@ -1117,19 +1117,23 @@ pub mod test {
             }
         }
 
+        assert_eq!(outbound_messages.len(), 1);
+        let messages = outbound_messages.clone();
+        let result = feedback_messages_with_errors(&mut coordinators, &mut signers, &messages);
```

Comment:
Is the point here to verify that the original `SignatureShareRequest` succeeds without mutation?

Reply:
Yup, show that things work as normal when things aren't modified, just as a sanity check.